go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	k8s.io/apimachinery v0.24.2
 	k8s.io/client-go v0.24.2
 	k8s.io/component-base v0.24.2
+	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
```

Comment:
❓ This moved from `// indirect` to direct. Does a `go mod tidy` revert this bit?

Reply:
Hmmm, no, it doesn't--curiously, when I start over from master to see what changed this line, it seems to be `go mod tidy`.

Reply:
I see what happened!

Reply:
Opened #3545.

Reply:
@tony-landreth re: our conversation